content/docs/user-guide/how-to/undo-adding-data.md

```diff
@@ -0,0 +1,46 @@
+# Undo Adding Data
+
+There are situations where you want to stop tracking data added previously.
+Follow the steps listed here to undo `dvc add`.
+
+Let's first add a data file into an example <abbr>project</abbr> using
+`dvc add`, which creates a `.dvc` file to track the data:
+
+```dvc
+$ dvc add data.csv
+$ ls
+data.csv    data.csv.dvc
+```
+
+Now let's reverse this action:
+
+> Note,if you are using `symlink` or `hardlink` as
+> [link](doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache)
+> type for DVC <abbr>cache</abbr>, you will have to unprotect the tracked file
+> first (see `dvc unprotect`):
+>
+> ```dvc
+> $ dvc unprotect data.csv
+> ```
+
+You'll need to remove the corresponding `.dvc` file and `.gitignore` entry using
+`dvc remove`:
```

Comment:
This sentence needs to be part of the previous (before block quote) or following (after code block) paragraphs.

Please always check how your work looks in a local development app to catch strange text organization like this.

Reply:
Yes missed this as earlier blockquote was a paragraph. Updated now.

Reply:
Looks good, thanks. I did a copy edit. Resolving